src/main/java/edu/harvard/iq/dataverse/pidproviders/FakePidProviderServiceBean.java

```diff
@@ -0,0 +1,66 @@
+package edu.harvard.iq.dataverse.pidproviders;
+
+import edu.harvard.iq.dataverse.AbstractGlobalIdServiceBean;
+import edu.harvard.iq.dataverse.DvObject;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.ejb.Stateless;
+
+@Stateless
+public class FakePidProviderServiceBean extends AbstractGlobalIdServiceBean {
+
+    @Override
+    public boolean alreadyExists(DvObject dvo) throws Exception {
+        return true;
```

Comment:
See my comment above.

This could be extended, too: when the fake provider would add a number to a simple database field, that could be used for constantly growing index number usable as a "real" faked PID. Otherwise things like UUIDs or similar could be used, which are garantied to be doublication free. It could also be an option to lookup the highest number in the database, increment and use. That is slow, but for demo / dev purposes this could be alright.